packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -18,9 +18,14 @@ interface FormatFileProps {
    * onDelete callback - this function will be called when the delete action is triggered
    */
   onDelete?(): void;
+  showMetaData?: boolean;
 }
 
-export function FormatFile({ file, onDelete }: FormatFileProps) {
+export function FormatFile({
+  file,
+  onDelete,
+  showMetaData = true,
```

Comment:
What is `showMetaData` doing here?

Reply:
When false, it would hide the name and size information that this components shows by default.